SmartDeviceLink/private/SDLVoiceCommandManager.m

```diff
@@ -115,8 +115,14 @@ - (void)setVoiceCommands:(NSArray<SDLVoiceCommand *> *)voiceCommands {
     // Set the new voice commands internally
     _voiceCommands = voiceCommands;
 
+    if (![self sdl_arePendingVoiceCommandsUnique:voiceCommands]) {
+        SDLLogE(@"Not all voice command strings are unique across all voice commands. Voice commands will not be set.");
+        return;
+    }
```

Comment:
This needs to be done before setting the voice commands in L116 or else it doesn't work properly.